nixGL.nix

```diff
@@ -15,7 +15,7 @@ nvidiaVersionFile ? null,
 enable32bits ? true
 , writeTextFile, shellcheck, pcre, runCommand, linuxPackages
 , fetchurl, lib, runtimeShell, bumblebee, libglvnd, vulkan-validation-layers
-, mesa, libvdpau-va-gl, intel-media-driver, vaapiIntel, pkgsi686Linux, driversi686Linux
+, mesa, libvdpau-va-gl, intel-media-driver, vaapiIntel, vaapiVdpau, pkgsi686Linux, driversi686Linux
```

Comment:
FYI intel-media-driver is preferred over vaapiIntel and vaapiVdpau. One of them is dead since 8 years and the other is for old hardware only.

Reply:
fixed, removed both drivers, it should be simple enought to create something like nixGLIntelOld for vaapiIntel if necessary.